packages/cli/src/types.ts

```diff
@@ -13,6 +13,21 @@ export interface ProjectSpecBase {
   endpoint: string;
 }
 
+export interface queryAD {
```

Comment:
We can name this more clearly. Interfaces should also start with a capital

```suggestion
export interface QueryAdvancedOpts {
```